PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -260,6 +264,10 @@ extension DefaultControlsViewController {
             }()
             
             contentFullScreenAction = props.player?.item.playable?.contentFullScreen ?? .nop
+            
+            isBrandedContentHidden = props.player?.item.playable?.brandedContent == nil
+            brandedContentTitle = props.player?.item.playable?.brandedContent?.advertisementText ?? ""
+            showBrandedContentWebSite = props.player?.item.playable?.brandedContent?.showWebPageCommand ?? .nop
```

Comment:
I think there should be both title and button in `uiProps` because if, for example, you will get BeOn video, but there won't be click URL, for some reason, you won't be able to hide this button.
It should be something like
```
showBrandedContentButtonHidden = props.player?.item.playable?.brandedContent?.showWebPageCommand == nil
```

Reply:
👍

Reply:
`showWebPageCommand` could be nil only if `brandedContent` nil, but in this case we should not show controls at all. So, in this implementation separate flags for title and button will be unnecessarily